app/main/browser.js

```diff
@@ -42,6 +44,12 @@ certs.setupCertificateHandlers();
 spawn.startContentService();
 spawn.startUserAgentService(userAgentClient);
 
+// Constant promise used to signal elsewhere in the main process
+// that the application has launched and finished loading
+// the initial browser window
+const appInitializedDeferred = new Deferred();
+app.INITIALIZED = appInitializedDeferred.promise;
```

Comment:
I don't like adding new bits of data to the electron API like this. Can we just expose this sort of thing in its own module?